components/Makefile

```
@@ -17,39 +17,41 @@ all: clean dist
 
 .PHONY: clean
 clean:
-	cd .. && yarn shx rm -rf storybook-static
+	cd .. && pnpm shx rm -rf storybook-static
 
 # artifacts
 #####################################################################
 
 .PHONY: dist
 dist:
-	cd .. && yarn sb build
+	cd .. && pnpm sb build
 
 .PHONY: lib
 lib: export NODE_ENV := production
 lib:
-	yarn vite build
+	pnpm vite build
 
 .PHONY: build-ts
 build-ts:
-	yarn tsc --build --emitDeclarationOnly
+	pnpm tsc --build --emitDeclarationOnly
 
 # development
 #####################################################################
 
 .PHONY: dev
 dev:
-	cd .. && yarn sb dev --port $(port)
+	# Run the storybook development server using pnpm
+	cd .. && pnpm sb dev --port $(port)
 
 .PHONY: test
 test:
 	$(MAKE) -C .. test-js-components tests="$(tests)" test_opts="$(test_opts)"
 
 .PHONY: test-cov
 test-cov:
+	$(MAKE) -C .. test-js-components tests=$(tests) test_opts="$(test_opts)" cov_opts="$(cov_opts)"
 	make -C .. test-js-components tests=$(tests) test_opts="$(test_opts)" cov_opts="$(cov_opts)"
```

This looks like an accidentally duplicated line?


```suggestion
	$(MAKE) -C .. test-js-components tests=$(tests) test_opts="$(test_opts)" cov_opts="$(cov_opts)"
```

---

removed